integration-tests/controlbus/src/test/java/org/apache/camel/quarkus/component/controlbus/it/ControlbusTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.controlbus.it;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.ExtractableResponse;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+@QuarkusTest
+class ControlbusTest {
+
+    @Test
+    public void test() {
+        ExtractableResponse response = RestAssured.given()
+                .contentType(ContentType.TEXT).get("/controlbus/status")
+                .then().extract();
+
+        String body = response.body().asString();
+        Assertions.assertEquals("Started", body);
```

Comment:
BTW you could make this shorter by using `.then().body(org.hamcrest.CoreMatchers.equalTo("Started"))`

Reply:
Thanks I have updated the test